src/api/app/components/action_seen_by_user_component.html.haml

```diff
@@ -0,0 +1,19 @@
+- if @render_only
+  - if seen_by_user
+    %i.fa-regular.fa-eye
```

Comment:
That sounds reasonable. I gave it a try with the eye but it makes more sense to keep it consistent.

Reply:
The only drawback here is that a checkbox looks like to be clickable but actually it is not, and I would not make it clickable in the dropdown, otherwise one can toggle it `as seen` even without loading it, just marking it from the dropdown. What do you think? is it clear enough? In the end if someone tries to click on the checkbox within the dropdown, it will loads the action and then he can marks it `as seen` from the page just loaded.